Makefile.toml

```diff
@@ -164,6 +164,7 @@ description = "Build WASM FFI into the target directory (release mode)"
 category = "ICU4X WASM"
 install_crate = { rustup_component_name = "rust-src" }
 toolchain = "nightly-2021-02-28"
+env = { "RUSTFLAGS" = "-C panic=abort -C opt-level=s" }
```

Comment:
Yeah, this is a tricky one. I guess in theory there should never be any panics, since any errors in user input should be handled by the result type (and safety concerns are handled by the high-level wrapper). But on the other hand you could end up with very confusing behavior if there are silent panics. I think for now we can keep panic logging disabled in release builds since they contribute quite a bit to the code size, but can reconsider if panics end up being more common than expected.

Reply:
That makes sense to me! Perhaps add a comment that "we don't care about panics in release mode because incorrect input is largely handled by result types"

Reply:
I would make it stronger; ICU4X should never panic, and therefore we do not want to carry around the significant code size associated with the panicking infrastructure.